examples/modal.py

```diff
@@ -8,7 +8,7 @@
 
 class MyModal(disnake.ui.Modal):
     def __init__(self) -> None:
-        components = [
+        components = disnake.ui.ModalActionRow(
```

Comment:
This doesn't quite work as action rows can only contain one of these components each, we can probably keep it as a list like before

Reply:
https://github.com/DisnakeDev/disnake/pull/558/commits/31d2e119bdf7c5597c4d3af2f41dc0e3d4f6eee3